pkg/karmadactl/deinit/deinit.go

```diff
@@ -22,6 +22,9 @@ const (
 )
 
 var (
+	deInitLong = templates.LongDesc(`
+		Removes the Karmada control plane from the Kubernetes cluster.`)
```

Comment:
I found that some places use the singular form and some places use the plural form.
How about removing the `s` uniformly?

Reply:
done